README.md

```diff
@@ -86,6 +86,20 @@ By default, the Rad Resource Resolver throws a `Symfony\Component\HttpKernel\Exc
             required: false
 ```
 
+## Configure the exception when the resource is not found
+
+By default, if the resource is required but null, the component will throw a 404 exception. You can customize this behavior by setting the `on-missing` parameter. Actually, 404, 401 and 403 are supported.
+
+```yaml
+    _resources:
+        buildings:
+            service: app.building.repository
+            method: findByCountryAndCityAndActivity
+            arguments: [$countryId, $citySlug, "School"]
+            on-missing:
+                throw: 401
```

Comment:
Would be cool to directly pass an exception FQCN

Reply:
@NiR- Sounds interesting. Would we be able to also configure the message exception, then?

Reply:
If we would support exception message, we would have to support other parameters I guess. I think it would be better to use a message defined by this component and indicating which resource resolution failed.